notebooks/official/tabular_workflows/bqml_arima_plus.ipynb

```diff
@@ -493,9 +461,9 @@
       "source": [
         "### Prepare training data in BigQuery\n",
         "\n",
-        "Before training a model, you must first generate our dataset of store sales. This dataset will include multiple products and stores, and it will also simulate factors such as advertisements and holiday effects. The data will be split into `TRAIN`, `VALIDATE`, `TEST`, and `PREDICT` sets, where the last three sets are all 1 month in duration.\n",
+        "Before training a model, you should first generate our dataset of store sales. This dataset includes multiple products and stores, and it also simulates factors such as advertisements and holiday effects. The data then be split into `TRAIN`, `VALIDATE`, `TEST`, and `PREDICT` sets, where the last three sets are all 1 month in duration.\n",
```

Comment:
- change: "The data then be split into ..."
- to: "The data is then split into ... "